script/foundry/deployment/Main.s.sol

```diff
@@ -13,6 +13,7 @@ contract Main is DeployHelper {
     uint256 internal CREATE3_DEFAULT_SEED = 0;
     // For arbitration policy
     uint256 internal constant ARBITRATION_PRICE = 1000 * 10 ** 6; // 1000 USDC
+    address internal constant TREASURY_ADDRESS = address(200);
```

Comment:
As the Main.s.sol is product deployment. Would be better read the `TREASURY_ADDRESS` from a environment variable as we did for multisig protocol admin, instead of harding code in the script?

Reply:
this could be in another pr - will create github issue